### ISSUE TITLE
pkce no longer appears in src/Provider/AbstractProvider.php

### ISSUE THREAD

**Issue by drennvinn** [thephpleague/oauth2-client#982]
Hello, I installed the project via composer (2.6.1) and started using it as described [here](https://oauth2-client.thephpleague.com/usage) but found out that AbstractProvider.php no longer had support for PKCE when I wanted to use 
```php
$provider = new \League\OAuth2\Client\Provider\GenericProvider([
    // ...
    // other options
    // ...
    'pkceMethod' => \League\OAuth2\Client\Provider\GenericProvider::PKCE_METHOD_S256
]);
```
So when i navigate to [https://github.com/thephpleague/oauth2-client/blob/master/src/Provider/AbstractProvider.php](https://github.com/thephpleague/oauth2-client/blob/master/src/Provider/AbstractProvider.php) i can found `setPkceCode()`, `getPkceCode()`, etc...  but not in the sources after composer install

**Comment by rhertogh** [comment]
@drennvinn As @megaxorg mentioned the latest changes aren't released yet.
In the meantime you can use [this workaround](https://github.com/thephpleague/oauth2-client/pull/901#issuecomment-1275357694).

**Comment by carlituxman** [comment]
Trying the PR [43c59dd](https://github.com/thephpleague/oauth2-client/commit/43c59dd35296f0fce59fc6726517c67e3d3be98c) it works.

**Comment by ramsey** [comment]
PKCE support is now available in version 2.7.0.
